Dockerfile.tpl

```diff
@@ -0,0 +1,12 @@
+# Dockerfile represents the container being use to run the driver, should be
```

Comment:
Please, remove all comments from the template and leave only any comment you need that is specific to your Dockerfile, or remove comments altogether.